QEfficient/utils/constants.py

```diff
@@ -10,7 +10,7 @@
 UTILS_DIR = os.path.dirname(os.path.abspath(__file__))
 QEFF_DIR = os.path.dirname(UTILS_DIR)
 ROOT_DIR = os.path.dirname(QEFF_DIR)
-QEFF_MODELS_DIR = os.path.join(ROOT_DIR, "qeff_models")
+QEFF_MODELS_DIR = os.environ.get("QEFF_MODELS_DIR", os.path.join(ROOT_DIR, "qeff_models"))

```

Comment:
You can pick this change from `release/v1.16` branch

Reply:
I have added the changes in the recent commit.